subsys/bluetooth/host/Kconfig

```diff
@@ -431,7 +431,6 @@ config BT_FIXED_PASSKEY
 
 config BT_USE_DEBUG_KEYS
 	bool "Enable Security Manager Debug Mode"
-	depends on BT_TINYCRYPT_ECC
```

Comment:
Should depend on BT_ECC now, no?

Reply:
Correct, should be handled since this is under `if BT_SMP`, which has `select BT_ECC`